modules/virt-virtctl-commands.adoc

```diff
@@ -54,4 +60,10 @@ the specified port of the node.
 
 |`virtctl image-upload dv <datavolume_name> --size=<datavolume_size> --image-path=</path/to/image>`
 |Upload a virtual machine image to a new DataVolume.
+
+|`virtctl version`
+|Print the client and server version information.
```

Comment:
I wonder if you should s/Print/Display to avoid ambiguity.

Reply:
Changed to Display.